src/indicator.js

```diff
@@ -201,13 +206,15 @@ var CPUFreqIndicator = class CPUFreqIndicator extends baseindicator.CPUFreqBaseI
         if (parseFloat(Config.PACKAGE_VERSION.substring(0,4)) > 3.32) {
             this.imSliderMin.add_child(this.minSlider);
         } else {
-            this.imSliderMin.actor.add(this.minSlider.actor, {expand: true});
```

Comment:
I changed this to be backwards compatible with Gnome 2.28 e.g. on Ubuntu 18.04 LTS. Otherwise, the extension crashes with the error, that `add()` wants a parameter of type `actor` but gets `undefined` from `Slider.actor`

Reply:
By the way ... d9beb is not that dirty. I've originally named it that way because I was afraid that it would break more than it fixes, but all of my tests worked perfectly fine ^^